docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.4'
+
+configs:
+  prometheus_config:
+    file: ./TodoApi/prometheus.yml
+
+services:
+  todoapi:
```

Comment:
Should this service and the container image be called "todo-api" so that it's consistent with "todo-web-server"?

Reply:
Yep. It's renamed.